cluster-management/setup/mounting-storage/index.md

```diff
@@ -108,6 +108,36 @@ coreos:
 
 Note the declaration of `ConditionPathExists=!/var/lib/docker.btrfs`. Without this line, systemd would reformat the btrfs filesystem every time the machine starts.
 
+## Mounting NFS shares
+
+This example will show how you can mount NFS share. First of all we will enable rpc-statd for NFS host monitoring. Then we will mount `/var/www` share into local `/var/www` path. Please pay attention on the unit name `var-www.mount` it should correspend to mount path `/var/www`.
+
+```yaml
+#cloud-config
+coreos:
+  units:
+    - name: rpc-statd.service
+      command: start
+      enable: true
+    - name: var-www.mount
+      command: start
+      content: |
+        [Mount]
+        What=nfs.example.com:/var/www
+        Where=/var/www
+        Type=nfs
+```
+
+If you would like to add NFS mount as dependency for certain unit you have to use these options:
+
+```yaml
+[Unit]
+After=var-www.mount
+Requires=var-www.mount
+```
+
+If mount point could not be mounted unit will not start.
```

Comment:
If **the** mount point **can't be** mounted **correctly**, **the dependent** unit will not start.